admin/includes/languages/english/index.php

```diff
@@ -80,39 +58,10 @@
 define('BOX_ENTRY_FEATURED_ACTIVE', 'Featured Products Active');
 define('BOX_ENTRY_SALEMAKER_ACTIVE', 'Sales Active');
 
-define('LAST_10_DAYS', 'Counter History for last %s recorded days');
 define('SESSION', 'Session');
 define('TOTAL', 'Total');
 
-// DASHBOARD - HOME PAGE OF ADMIN - CUSTOMERS section
-define('BOX_TITLE_CUSTOMERS', 'Customers');
-define('BOX_ENTRY_CUSTOMERS_NORMAL', '- Full Accounts :');
-define('BOX_ENTRY_CUSTOMERS_TOTAL', 'Total Customer Accounts :');
-define('BOX_ENTRY_CUSTOMERS_TOTAL_DISTINCT', 'Total Distinct Customers :');
-
-define('TEXT_REMOVE', 'Remove');
 define('TEXT_UPDATE', 'Update');
-define('TEXT_CONFIRM_REMOVE', 'Are you sure you want to remove this widget from the dashboard?');
-define('TEXT_WIDGET_UPDATE_HEADER', 'Widget Settings have been updated');
-
-define('TEXT_TIMER_SELECT_NONE', 'No Refresh');
-define('TEXT_TIMER_SELECT_1MIN', 'Refresh every Minute');
-define('TEXT_TIMER_SELECT_5MIN', 'Refresh every 5 Minutes');
-define('TEXT_TIMER_SELECT_10MIN', 'Refresh every 10 Minutes');
-define('TEXT_TIMER_SELECT_15MIN', 'Refresh every 15 Minutes');
-
-define('TEXT_NO_WIDGETS_TO_INSTALL', 'There are currently no widgets available to install.');
-define('TEXT_FORM_ERROR_CHOOSE_ZONE', 'Please choose a Zone');
-define('TEXT_WARNING_SUPERUSER_REQUIRED', 'Only a Superuser can complete the initial setup. You will not be able to use your Admin until this is done.');
-define('TEXT_HEADING_SETUP_WIZARD', 'Initial Setup Wizard');

```

Comment:
really? aren't these used in the setup page?

Reply:
Nope. `ff` is an alias for `find . -type f`

Scotts-iMac-2020:gh_demo_157 scott$ grep TEXT_TIMER_SELECT_NONE `ff`
./admin/includes/languages/english/index.php:define('TEXT_TIMER_SELECT_NONE', 'No Refresh');
Scotts-iMac-2020:gh_demo_157 scott$ grep TEXT_WARNING_SUPERUSER_REQUIRED `ff`
./admin/includes/languages/english/index.php:define('TEXT_WARNING_SUPERUSER_REQUIRED', 'Only a Superuser can complete the initial setup. You will not be able to use your Admin until this is done.');
Scotts-iMac-2020:gh_demo_157 scott$ grep TEXT_HEADING_SETUP_WIZARD `ff`
./admin/includes/languages/english/index.php:define('TEXT_HEADING_SETUP_WIZARD', 'Initial Setup Wizard');
Scotts-iMac-2020:gh_demo_157 scott$ grep TEXT_NO_WIDGETS_TO_INSTALL `ff`
./admin/includes/languages/english/index.php:define('TEXT_NO_WIDGETS_TO_INSTALL', 'There are currently no widgets available to install.');
Scotts-iMac-2020:gh_demo_157 scott$ grep TEXT_FORM_ERROR_CHOOSE_ZONE `ff`
./admin/includes/languages/english/index.php:define('TEXT_FORM_ERROR_CHOOSE_ZONE', 'Please choose a Zone');